wrappers/winuser.go

```diff
@@ -106,6 +106,18 @@ const (
 	DF_ALLOWOTHERACCOUNTHOOK = 0x00000001
 )
 
+const (
+	EWX_LOGOFF          = 0
+	EWX_SHUTDOWN        = 0x00000001
+	EWX_REBOOT          = 0x00000002
+	EWX_FORCE           = 0x00000004
+	EWX_POWEROFF        = 0x00000008
+	EWX_FORCEIFHUNG     = 0x00000010
+	EWX_QUICKRESOLVE    = 0x00000020
```

Comment:
`EWX_QUICKRESOLVE` also appears to be undocumented.  See https://docs.microsoft.com/en-us/windows/desktop/api/winuser/nf-winuser-exitwindowsex.